ChangeLog.rst

```diff
@@ -9,6 +9,94 @@ Unreleased
   releasing next version, the "Unreleased" header will be replaced with
   appropriate version header and this help text will be removed.
 
+Version 2.0.0
+-------------
```

Comment:
It's a cruft that just helps out writing the changelog while developing, but it shouldn't be there to distract downloaders of the release. (It's not intended for them.)